README.md

```diff
@@ -4,25 +4,26 @@
 
 Try out these Zilla example configurations using `docker` in your local development enivronment.
 
-| Name | Description|
-|------|------------|
-| [tcp.echo](tcp.echo) | Echoes bytes sent to the TCP server |
-| [tcp.reflect](tcp.reflect) | Echoes bytes sent to the TCP server, broadcasting to all TCP clients |
-| [tls.echo](tls.echo) | Echoes encrypted bytes sent to the TLS server |
-| [tls.reflect](tls.reflect) | Echoes encrypted bytes sent to the TLS server, broadcasting to all TLS clients |
-| [http.filesystem](http.filesystem) | Serves files from a directory on the local filesystem |
-| [http.echo](http.echo) | Echoes request sent to the HTTP server from an HTTP client |
-| [http.echo.jwt](http.echo.jwt) | Echoes request sent to the HTTP server from a JWT-authorized HTTP client |
-| [http.kafka.sync](http.kafka.sync) | Correlates requests and responses over separate Kafka topics |
-| [http.kafka.async](http.kafka.async) | Correlates requests and responses over separate Kafka topics, asynchronously |
-| [http.kafka.cache](http.kafka.cache) | Serves cached responses from a Kafka topic, detect when updated |
-| [http.kafka.oneway](http.kafka.oneway) | Sends messages to a Kafka topic, fire-and-forget |
-| [amqp.reflect](amqp.reflect) | Echoes messages published to the AMQP server, broadcasting to all receiving AMQP clients |
-| [mqtt.reflect](mqtt.reflect) | Echoes messages published to the MQTT server, broadcasting to all receiving MQTT clients |
-| [sse.kafka.fanout](sse.kafka.fanout) | Streams messages published to a Kafka topic, applying conflation based on log compaction |
-| [sse.proxy.jwt](sse.proxy.jwt) | Proxies messages delivered by the SSE server, enforcing streaming security constraints |
-| [ws.echo](ws.echo) | Echoes messages sent to the WebSocket server |
-| [ws.reflect](ws.reflect) | Echoes messages sent to the WebSocket server, broadcasting to all WebSocket clients |
```

Comment:
Good job adding the new example to the table.
Please revert the spacing changes, no need for vertical alignment in raw markdown to get vertical alignment in rendered output.

Reply:
done.